### PR TITLE
chore: add keypoint detection model output

### PR DESCRIPTION
**Pull request by Phelan164**
Because

- Support Keypoint task

This commit

- Keypoint detection model output

### CODE REVIEW

File: vdp/model/v1alpha/model.proto [hunk 1]
```diff
@@ -500,6 +502,29 @@ message ClassificationOutputs {
...
+  repeated Keypoint keypoints = 1 [ (google.api.field_behavior) = OUTPUT_ONLY ];
+  // Keypoint score
+  float score = 2 [ (google.api.field_behavior) = OUTPUT_ONLY ];
+}
+
+// KeypointOutput represents the output of keypoint detection task
+// The keypoint detection model do not support batch, so output is singular
+message KeypointOutput {
+  // Keypoint Objects
+  repeated KeypointObject keypoint_output = 1 [ (google.api.field_behavior) = OUTPUT_ONLY ];
```
Comment (xiaofei-du) [inline vdp/model/v1alpha/model.proto:525]: Should we change `keypoint_output` to `keypoint_objects`?

Comment (Phelan164) [reply vdp/model/v1alpha/model.proto:525]: @xiaofei-du we could but I just keep consistency with classification/detection
repeated DetectionOutput detection_outputs = 1
repeated ClassificationOutput classification_outputs = 1

Comment (Phelan164) [reply vdp/model/v1alpha/model.proto:525]: btw. the keypoint detection model do not support batch so I put output as singular

Comment (pinglin) [reply vdp/model/v1alpha/model.proto:525]: The API is for general cases. Naming convention-wise, it should be `keypoint_outputs`.

Comment (xiaofei-du) [reply vdp/model/v1alpha/model.proto:525]: ok, got it.

**Pull request merged**